data/kubemacpool/kubemacpool.yaml

```diff
@@ -1,9 +1,6 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  labels:
-    control-plane: mac-controller-manager
```

Comment:
why `control-plane: mac-controller-manager` is removed as well ?
it was not part of the latest breaking change

Reply:
+1, it should stay. I think we already delete it for non-namespaced objects ([here](https://github.com/kubevirt/cluster-network-addons-operator/blob/main/pkg/controller/networkaddonsconfig/networkaddonsconfig_controller.go#L537)).

Reply:
Btw some of the logic there was in order to maintain backward compatability
maybe the versions we support /deployed on customer site are new enough so we can remove this patch?
(more info here https://github.com/kubevirt/cluster-network-addons-operator/pull/1145)

Reply:
I would not depend on the controller and keep the manifest as we want them to be, so we don't need time for them to be fixed.

Reply:
The controller is removing them only for backward compatibility, for versions where CNAO label stuff that it didn't need to have and then those versions were upgraded to one that fixed the labeling.

You can see here a discussion that tells that we better remove it from the source itself - KMP, IIRC
https://github.com/kubevirt/cluster-network-addons-operator/pull/1145/files#r796372158

Reply:
need to rerun bump right ? since you changed the script